### PR TITLE
Update schema for "AllowInterruptions"

### PR DESCRIPTION
**Pull request by boydc2014**

### CODE REVIEW

**Comment by boydc2014** [comment]
I did this manually, @chrimc62 do you have a guidance how can i do this use dialogSchema?

File: schemas/sdk.schema [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "$schema": "https://raw.githubusercontent.com/Microsoft/botbuilder-dotnet/4.Future/Schemas/component.schema",
+    "$schema": "https://raw.githubusercontent.com/Microsoft/botbuilder-tools/master/packages/DialogSchema/src/dialogSchema.schema",
```
Comment (tomlm) [inline schemas/sdk.schema]: >"https://raw.githubusercontent.com/Microsoft/botbuilder-tools/master/packages/DialogSchema/src/dialogSchema.schema", [](start = 15, length = 116)

I don't think this is supposed to change....

Comment (chrimc62) [reply schemas/sdk.schema]: The botbuilder-dotnet path is the correct one.  The other one doesn't even exist...  This change is likely caused by using an older dialogschema tool, not the most recent one.

Comment (boydc2014) [reply schemas/sdk.schema]: I reverted this line, but seems the latest dialogSchema is generating this  see https://github.com/microsoft/botbuilder-tools/blob/44dc9e87e18e80831a260c555e0d4e5cd1ffc3bd/packages/DialogSchema/src/dialogSchema.schema#L3

**Pull request merged**